fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/SkolemNodeRdfContext.java

```diff
@@ -93,6 +94,10 @@ public SkolemNodeRdfContext(final FedoraResource resource,
             return null;
 
         } catch (final RepositoryException e) {
+            if (e instanceof ItemNotFoundException) {
+                // Effectively skip resources that do not exist
```

Comment:
Indeed, one should not be able to create links to non-existent resources... you might even consider it a bug. Actually, there is a ticket for that bug:
https://jira.duraspace.org/browse/FCREPO-2323

This PR is the resolution.